src/items.rs

```diff
@@ -806,8 +804,7 @@ fn rewrite_explicit_self(explicit_self: &ast::ExplicitSelf, args: &[ast::Arg]) -
 
             // this hacky solution caused by absence of `Mutability` in `SelfValue`.
             let mut_str = {
-                if let ast::Pat_::PatIdent(ast::BindingMode::BindByValue(mutability), _, _)
-                        = args[0].pat.node {
+                if let ast::Pat_::PatIdent(ast::BindingMode::BindByValue(mutability), _, _) = args[0].pat.node {
```

Comment:
Right. I think it's the same issue as the one in the assignment PR: we don't format indexation or properties. There should be a fix in the assignment PR for this now.